packages/cactus-plugin-ledger-connector-ethereum/src/main/typescript/web-services/deploy-contract-v1-endpoint.ts

```diff
@@ -88,6 +128,17 @@ export class DeployContractEndpoint implements IWebServiceEndpoint {
   public async handleRequest(req: Request, res: Response): Promise<void> {
     const reqTag = `${this.getVerbLowerCase()} - ${this.getPath()}`;
     this.log.debug(reqTag);
+    const isValid = validateDeployContract(req.body);
+    if (!isValid) {
+      this.log.warn(
```

Comment:
@Harshdev098 Please update this to log on the debug level instead. While in a low traffic/dev environment the warn log level might help bringing this to the attention of the user quickly, in production it will just spam the logs with warnings that are not actionable to an ops person because an invalid incoming request doesn't imply that there is a bug to be fixed at all.